### DISCUSSION TITLE
How to change default cell size/zoom?

### DISCUSSION THREAD

**Discussion by kirawi** [wezterm/wezterm#1230] [Q&A] [answered]
Right now, it's way too big:
![Capture](https://user-images.githubusercontent.com/67773714/137516597-31750bc0-806d-40fb-a7ec-796b1d3a4364.PNG)

**Comment by bew** [comment, marked as answer]
You can configure the default font size with [`font_size`](https://wezfurlong.org/wezterm/config/lua/config/font_size.html) in your [configuration file](https://wezfurlong.org/wezterm/config/files.html).
For example:
```lua
return {
  font_size = 8.0,
}
```

**Reply by kirawi** [reply]
It also looked like I placed the config file in the wrong directory.
